sematic/ui/src/pipelines/NotesPanel.tsx

```diff
@@ -37,6 +37,7 @@ export default function NotesPanel() {
   );
 
   const anonymousUser: User = useMemo(() => ({
```

Comment:
The `User` definition from `Models.tsx` must be updated with the new field.

Reply:
Saw this is done here: https://github.com/sematic-ai/sematic/pull/599/files#diff-899d28e8f89ca40d3c6fbfc5dd193fffe4288421bc83beefaae582036dc68b04